posts/templates/posts/post_detail.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-xs-0 col-sm-3 col-md-4"></div>
```

Comment:
Use the offset classes in the next div and don't put this in front.
Take a look at the bootstrap documentation. http://getbootstrap.com/css/#grid-offsetting